### PR TITLE
Nm 184

### PR DESCRIPTION
**Pull request by KyptoDaydream**

### CODE REVIEW

File: src/components/Team/collaboration/Collaboration.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              changes and help ensure that the changes meet the repository's contributing guidelines and other quality
+              standards.
+            </p>
+          </div>
+        </InfoTooltip>
+      </div>
+    );
+  }
+
+  private getTrend(sprints: I.GitIntervals[]): any {
```
Comment (dullus) [inline src/components/Team/collaboration/Collaboration.tsx:70]: remove :any, return value is inherited from parseTrendStandard

File: src/components/Team/collaboration/Collaboration.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+export interface IProps {
+  git?: I.StateGit;
+}
+
+export class Collaboration extends React.Component<IProps> {
+  public render() {
+    if (!this.props.git || this.props.git.intervals.length === 0) {
+      return <EmptyGraph />;
+    } else {
+      // const nonSprintTimeSpent = this.getAverageNonSprintTime(this.props.sprints);
```
Comment (lenikasss) [inline src/components/Team/collaboration/Collaboration.tsx:22]: is this needed?

File: src/components/Team/collaboration/collaborationChart/CollaborationChart.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+export function CollaborationChart(props: IProps): JSX.Element {
+  return (
+    <div className={style.collaborationChart}>
+      <LinearChart {...getQualityData(props.git.intervals)} />
+    </div>
+  );
+}
+
+function getQualityData(data: I.GitIntervals[]): IChart {
```
Comment (lenikasss) [inline src/components/Team/collaboration/collaborationChart/CollaborationChart.tsx:22]: maybe different name :)

File: src/components/Team/collaboration/Collaboration.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+  private getTitle(): JSX.Element {
+    return (
+      <div>
+        Collaboration
+        <InfoTooltip>
+          <div>
+            <p>
+              Pull requests comments are considered a good practice where reviews allow for discussion of proposed
+              changes and help ensure that the changes meet the repository's contributing guidelines and other quality
```
Comment (lukasorgovan) [inline src/components/Team/collaboration/Collaboration.tsx:61]: should be "contribution guilelines" not contributing

File: src/components/Team/collaboration/Collaboration.tsx (hunk 1 above)
Comment (KyptoDaydream) [reply src/components/Team/collaboration/Collaboration.tsx:70]: ill create ticket for this - need to change it across all charts and unify & move this function to utils

**Pull request merged**